odoo/addons/base/models/res_users.py

```diff
@@ -697,9 +697,11 @@ def context_get(self):
 
         return frozendict(context)
 
-    @tools.ormcache('self._uid')
+    @tools.ormcache('self.id')
```

Comment:
Hello @odony ,

It doesn't match the domain below (`self.id !=  self._uid`). For now, it doesn't create any issue because we call it only with the `env` user.
Did you forget to take in account your comment : https://github.com/odoo/odoo/pull/102586#discussion_r990832167  or I miss something ?

Reply:
yup I messed up, somehow my cherry-pick ended-up with the wrong version of the code 🤦
going to fix that up, it's definitely wrong, thanks!

Reply:
-> #102819